code/clue.js

```diff
@@ -53,7 +178,46 @@ const pickKiller = () => {
   ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
 }
 
+document.getElementById('killerCard').addEventListener('click', pickKiller)
+
+
+// This function will be invoked when you click on the weapon card.
+const pickWeapon = () => {
+  // This will randomly select a weapon. And add that to the mystery object.
+  mystery.weapon = randomSelector(weapons)
+
+  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
+  document.getElementById('weaponCard').style.background = mystery.killer.color
```

Comment:
I like the styling perspective on using killer.color for the weapon and room card, though this makes the weapon and room card unclickable before killer is picked (as killer.color is not defined until then). 
There are different approaches to avoid this: tell the user to start with the killer card, or use no or other colors than killer.color for the weapon and room cards.